src/main/webapp/donation.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FeedOn - Donation Form</title>
+
+    <!-- Favicon, CSS & JS -->
+    <link rel="icon" href="assets/logo.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/donation-form.css">
+    <script src="js/client.js" defer></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
+
+    <!-- Bootstrap CDN -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+
+<body onload="getBlobstoreURL()">
+
+    <form id="donation-form" class="form-container" method="POST" enctype="multipart/form-data">
```

Comment:
Which servlet would this post request be sent to?

Reply:
This first invokes the `BlobstoreUploadURLServlet`, which handles the processing of an image uploaded by the user in the donation form and returns a URL to access that image. This servlet in turn invokes the `DonationOfferServlet`which makes a POST request with the image URL returned by Blobstore. Hope that makes sense!